src/window.jl

```diff
@@ -106,6 +116,15 @@ body(win::Window, html::String) =
 const initcss = """
   <style>html,body{margin:0;padding:0;border:0;text-align:center;}</style>
   """
+body(win::Window, x) =
+  # FIXME: use tohtml
```

Comment:
@one-more-minute I cannot use BlinkDisplay here (not exported and not part of the Blink module). Should tohtml be in Blink instead? or do you see it going into Base?